Storage/SQL/SQLiteHistory.swift

```diff
@@ -71,7 +71,7 @@ func getLocalFrecencySQL() -> String {
     let microsecondsPerDay = 86_400_000_000.0      // 1000 * 1000 * 60 * 60 * 24
     let ageDays = "((\(now) - localVisitDate) / \(microsecondsPerDay))"
 
-    return "\(visitCountExpression) * max(2, 100 * 225 / (\(ageDays) * \(ageDays) + 225))"
+    return "\(visitCountExpression) * MAX(2, 100 * 225 / (\(ageDays) * \(ageDays) + 225))"
```

Comment:
Don't change the case. We uppercase keywords like `SELECT` and operators like `CASE` and `NOT`, but not function names and aggregate operations. This follows [SQLite's style](https://sqlite.org/lang_corefunc.html#maxoreunc).

Reply:
Ah, ok. I wasn't aware that lower-case functions were part of a styling convention. I'll go through and fix these.